### PR TITLE
[FREELDR] i386vid.c: Improve WIP r73587 documentation and code

### PR DESCRIPTION
**Pull request by SergeGautherie**
## Purpose

As this code is WIP and untouched for a year, do some simple cleanup.

## Proposed changes

- Improve/Add documentation.
- Improve code a bit.

### CODE REVIEW

File: boot/freeldr/freeldr/arch/i386/i386vid.c [hunk 1]
```diff
@@ -205,7 +204,8 @@ USHORT BiosIsVesaSupported(VOID)
 
     if (Regs.w.ax != 0x004F)
     {
-        ERR("VESA BIOS call failed\n");
+        WARN("VBE Get SuperVGA information function not supported (AL=0x%x) or failed (AH=0x%x)\n",
```
Comment (ThFabba) [inline boot/freeldr/freeldr/arch/i386/i386vid.c]: These trace changes look fine. All the unrelated noise around it does not.

**Comment by learn-more** [comment]
> As this code is WIP and untouched for a year, do some simple cleanup too.
Is it your mission to touch as much untouched code as possible?

File: boot/freeldr/freeldr/arch/i386/i386vid.c (hunk 1 above)
Comment (SergeGautherie) [reply boot/freeldr/freeldr/arch/i386/i386vid.c]: Trace changes splitted to #291.

**Comment by SergeGautherie** [comment]
> Is it your mission to touch as much untouched code as possible?

Assume I care about these changes.
What is the process to submit them? This PR, as is? Directly to @EricKohl (How?)?  ...? No way, stalled code?

**Comment by gigaherz** [comment]
Uhm... We just don't like "meaningless" edits in the code, because they make the patches (PR diffs) longer to read through, and they change the blame information that would otherwise be used to know who was the original coder.
This is specially true in files that no one has touched in a long time, because it means no one needs those edits to happen. 
So the idea is: if you aren't actually doing a meaningful change, don't touch the code. And if you are doing meaningful changes, don't touch unrelated code that will make it harder to review your real changes.

**Comment by binarymaster** [comment]
> Directly to EricKohl (How?)

Using e-mail.

**Comment by sanchaez** [comment]
Closed as being superfluous.

**Pull request closed without merging**